### ISSUE TITLE
How to align your images?

### ISSUE THREAD

**Issue by oliver-batchelor** [meyerls/FruitNeRF#1]
I tried with COLMAP to align your dataset first one by one, then secondly all images at once - they're 3 trees in a row right? But struggled to get it to do anything useful. I'm curious what settings you used?

You already have posted camera extrinsics - however I wanted the sparse point-cloud too.
I also tried with Metashape and had better success (though still quite a few images failed to align).

**Comment by meyerls** [comment]
Yes indeed there are 3 trees in a row. The original plan was that the trees should be considered individually. I was using handcrafted transformation matrices to align all 3 trees. So I would recommend using the provided transform.json for good results. 

If you want the sparse pcd you should try COLMAP on every tree individually and fuse them afterwards. If you are interested in the manual alignment and to fuse the extrinsics, please let me know.

**Comment by oliver-batchelor** [comment]
Thanks! I will try a few things.

We've been working on very similar ideas. We also have a dataset of apple fruitlets (multiple captures taken from a robot arm) that is quite similar in some ways. We could share it if it is useful to you—we probably just need to filter out the worst captures.

**Comment by Double-zh** [comment]
> 是的，确实有 3 棵树排成一排。最初的计划是应该单独考虑这些树。我使用手工制作的变换矩阵来对齐所有 3 棵树。因此，我建议使用提供的 transform.json 来获得良好的结果。
> 
> 如果您想要稀疏 pcd，您应该在每棵树上分别尝试 COLMAP，然后将它们融合。如果您对手动对齐和融合外部参数感兴趣，请告诉我。

Hello, how do I achieve the fusion of multiple fruit trees and the acquisition of camera external parameters? Can you explain the specific implementation process in detail or share the code?
